wrappers/ios/ci/build.sh

```diff
@@ -227,79 +225,44 @@ copy_lib_tocombine() {
     ARCHS="arm64 x86_64"
 
     for ARCH in ${ARCHS[*]}; do
-        cp -v $OUTPUT_DIR/libs/$LIB_NAME/$ARCH/$LIB_FILE_NAME.a $OUTPUT_DIR/cache/arch_libs/${LIB_FILE_NAME}_$ARCH.a
+        cp -v "$OUTPUT_DIR/libs/$LIB_NAME/$ARCH/$LIB_FILE_NAME.a" "$OUTPUT_DIR/cache/arch_libs/${LIB_FILE_NAME}_$ARCH.a"
```

Comment:
It's best practice to always enclose strings containing variable interpolation into double quotes, otherwise it can cause undesired behaviour if the variable contains whitespace (what you expected to be 1 literal token might become 2 or more separate tokens). That's most likely won't be the case here even in the future, but it is best practice.

Reply:
OK